### PR TITLE
Continue cleanup

### PR DESCRIPTION
**Pull request by JavierJia**
Fix some script parameters for the ease of changing the AsterixDB URL. 
Reduce the sample tweets to 47,000 records to speed up the quick start performance. And since the size is small, we store it directly in github.

### CODE REVIEW

File: docs/quick-start.md [hunk 1]
```diff
@@ -80,17 +80,24 @@ The instructions above assume that we use an AsterixDB instance in a Docker cont
 
 **Step 8**: Follow the instructions on the [AsterixDB Installation Guide](https://ci.apache.org/projects/asterixdb/index.html) to install an AsterixDB cluster.  Select your preferred installation option. 
 
-**Step 9**: Ingest twitter data.
+**Step 9**: Ingest twitter data to AsterixDB
 
-**Step 10**: Change the Cloudberry middleware configuration to connect to this new AsterixDB cluster. You can modify the AsterixDB hostname in the configuration file `neo/conf/application.conf` and change the `asterixdb.url` value to the AsterixDB hostname.
+You need to give the RESTFul API link of the AsterixDB cluster and one of its NC name to the ingestion script as following:
```
Comment (chenlica) [inline docs/quick-start.md]: "one of its NC name" -> "one of its NC names"

File: docs/quick-start.md [hunk 2]
```diff
@@ -80,17 +80,24 @@ The instructions above assume that we use an AsterixDB instance in a Docker cont
...
+
+```
+~/cloudberry> ./script/ingestAllTwitterToLocalCluster.sh http://YourAsterixDBServerIP:19002/aql ONE_OF_NC_NAME
```
Comment (chenlica) [inline docs/quick-start.md]: "ONE_OF_NC_NAME" -> "ONE_OF_NC_NAMES"

File: docs/quick-start.md [hunk 3]
```diff
@@ -80,17 +80,24 @@ The instructions above assume that we use an AsterixDB instance in a Docker cont
...
+You can modify the AsterixDB hostname in the configuration file `neo/conf/application.conf` by changing the `asterixdb.url` value.
 
 ```
 asterixdb.url = "http://YourAsterixDBHostName:19002/query/service"
 ```
 
 ## Build your own application
 
-For more information about Cloudberry, please read its [documentation](/documentation).
+TwitterMap is one example of how to use Cloudberry. To develop your own front-end application, please find more information in [documentation](/documentation).
```
Comment (chenlica) [inline docs/quick-start.md]: I still don't want to call the system a "frontend application."  Remove "front-end".

File: docs/quick-start.md (hunk 1 above)
Comment (JavierJia) [inline docs/quick-start.md]: done

File: docs/quick-start.md (hunk 2 above)
Comment (JavierJia) [inline docs/quick-start.md]: done

File: docs/quick-start.md (hunk 3 above)
Comment (JavierJia) [inline docs/quick-start.md]: got it. :-)

**Pull request merged**